src/DotNest.Core.SDK.Web/DotNest.Core.SDK.Web.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+    </PropertyGroup>
+  
+    <ItemGroup>
+      <PackageReference Include="Lombiq.Analyzers" Version="2.1.1-alpha.0" />
+      <PackageReference Include="OrchardCore.Application.Cms.Core.Targets" Version="1.4.0" />
+      <PackageReference Include="OrchardCore.Logging.NLog" Version="1.4.0" />
+    </ItemGroup>
+  
+    <ItemGroup>
+      <PackageReference Include="Lombiq.HelpfulExtensions" Version="4.1.0-alpha.1" />
+      <PackageReference Include="Lombiq.HelpfulLibraries" Version="4.5.0" />
+      <PackageReference Include="Lombiq.Hosting.Azure.ApplicationInsights" Version="3.1.0" />
+      <PackageReference Include="Lombiq.Hosting.Tenants.Admin.Login" Version="2.1.0" />
+      <PackageReference Include="Lombiq.Hosting.Tenants.Management" Version="2.1.0" />
+      <PackageReference Include="Lombiq.Privacy" Version="3.1.1" />
+      <PackageReference Include="Lombiq.SetupExtensions" Version="2.1.0" />
+      <PackageReference Include="Lombiq.Tests.UI.AppExtensions" Version="3.2.0" />
+      <PackageReference Include="Lombiq.Tests.UI.Shortcuts" Version="3.2.0" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Theme\Lombiq.BaseTheme\Lombiq.BaseTheme.csproj" />
```

Comment:
There is a nuget package for the Lombiq.BaseTheme, please use that. I'll continue the review once it's done.